### PR TITLE
Lit6203 malkea seqlat

### PR DESCRIPTION
**Pull request by MarcinKrawczuk**
This is a malk poem from EMML 1939.

### CODE REVIEW

File: new/LIT6203MalkeaSeqlat.xml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        <revisionDesc>
+            <change who="MKr" when="2020-10-19">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+          <div type="bibliography"/>
+
+          <div type="edition">
+            <div type="textpart" subtype="incipit">
```
Comment (DariaElagina) [inline new/LIT6203MalkeaSeqlat.xml:58]: you could add xml:lanf either to ab or to div

File: new/LIT6203MalkeaSeqlat.xml [hunk 2]
```diff
@@ -0,0 +1,67 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6203MalkeaSeqlat" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">መልክአ፡ ስቅለት፡</title>
+                <title xml:lang="gez" corresp="*t1">Malkǝʾa Sǝqlat</title>
```
Comment (thea-m) [inline new/LIT6203MalkeaSeqlat.xml:9]: sǝqlat with a small s. Normally, only the first word of work titles is spelled with a capital, except for proper names

File: new/LIT6203MalkeaSeqlat.xml [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        </profileDesc>
+        <revisionDesc>
+            <change who="MKr" when="2020-10-19">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+          <div type="bibliography"/>
+
+          <div xml:lang="gez" type="edition">
```
Comment (thea-m) [inline new/LIT6203MalkeaSeqlat.xml:57]: It is always better to indicate explicitly in the record from which source the text has been taken. You can write simply a `<note>` in edition with something like "This text is taken from MS xy"

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you, two minor remarks

**Review by thea-m — APPROVED** [review]

**Pull request merged**